colorcoder.h

```diff
@@ -0,0 +1,31 @@
+#pragma once
+#include<iostream>
+#include <string>
+
+namespace TelCoColorCoder
+{
+    enum class MajorColor {WHITE, RED, BLACK, YELLOW, VIOLET};
+    enum class MinorColor {BLUE, ORANGE, GREEN, BROWN, SLATE};
+
+    extern const char* MajorColorNames[];
+    extern int numberOfMajorColors;
+    extern const char* MinorColorNames[];
+    extern int numberOfMinorColors;
+
+    class ColorPair {

```

Comment:
Good approach to keep the `ColorPair` together with rest of the functionality in the same namespace.
However, keeping all the functionality in the same cpp may mean that multiple developers find it difficult to work on adding functionality simultaneously.